sdk/typespec/typespec_client_core/src/time/mod.rs

```diff
@@ -121,22 +120,22 @@ pub fn to_last_state_change(date: &OffsetDateTime) -> String {
 
 /// Create a duration from the number of minutes.
 pub fn duration_from_minutes(minutes: u64) -> Duration {
-    Duration::from_secs(minutes * 60)
+    Duration::minutes(i64::try_from(minutes).unwrap_or(i64::MAX))
```

Comment:
Given that `minutes` (and other values below) are unsigned, I ponder we should actually just `unwrap()` here and panic if it fails. Defaulting to `i64::MAX` for, example, days could be catastrophic! Better to fail than hang until the heat death of our solar system.

Reply:
I was wondering the same thing, TBH. I'll change it to be .unwrap().